src/plugins/elasticsearch/index.js

```diff
@@ -19,7 +19,7 @@ module.exports = function ({ Plugin }) {
         username: string(),
         password: string(),
         shardTimeout: number().default(0),
-        requestTimeout: number().default(30000),
+        requestTimeout: number().default(300000),
```

Comment:
I put this in so it matched the default value in the kibana.yml config file. Agreed, 5 minutes is probably a bit excessive.